docs/example/README.md

```diff
@@ -28,65 +42,142 @@ Use the [installation guide](/docs/installation_guide.md) to install the `osm` c
+Position the windows so that you can see all four at the same time. The header at the top of the webpage indicates the application and version. 
 
-## Access Control Policies
+## Deploy SMI Access Control Policies for OSM
 At this point, no applications have access to each other because no access control policies have been applied. Confirm this by confirming that none of the counters in the UI are incrementing. Apply the [SMI Traffic Target][1] and [SMI Traffic Specs][2] resources to define access control policies below:
 ```bash
-kubectl apply -f docs/example/manifests/access/
+kubectl create -f docs/example/manifests/access/
 ```
-The counters should now be incrementing for the windows with headers: `Bookbuyer`, and `Bookstore-v1`.
+The counters should now be incrementing for the `Bookbuyer`, and `Bookstore-v1` applications:
+- http://localhost:8080 - **Bookbuyer**
+- http://localhost:8081 - **bookstore-v1**
 
-### Going Further
-Uncomment out the lines in the [manifests/access/traffic-access.yaml](manifests/access/traffic-access.yaml) to allow `Bookthief` to communicate with `Bookstore`. Then, re-apply the manifest and watch the change in policy propagate.
+### Allowing the Bookthief Application to access the Mesh
+Currently the Bookthief application has not been authorized to participate in the service mesh communication. We will now uncomment out the lines in the [docs/example/manifests/access/traffic-access.yaml](manifests/access/traffic-access.yaml) to allow `Bookthief` to communicate with `Bookstore`. Then, re-apply the manifest and watch the change in policy propagate.
+
+Current TrafficTarget spec with commented `Bookthief` kind:
+```
+kind: TrafficTarget
+apiVersion: access.smi-spec.io/v1alpha2
+metadata:
+  name: bookstore-v1
+  namespace: bookstore
+spec:
+  destination:
+    kind: ServiceAccount
+    name: bookstore-v1
+    namespace: bookstore
+  rules:
+  - kind: HTTPRouteGroup
+    name: bookstore-service-routes
+    matches:
+    - buy-a-book
+    - books-bought
+  sources:
+  - kind: ServiceAccount
+    name: bookbuyer
+    namespace: bookbuyer
+  #- kind: ServiceAccount
+    #name: bookthief
+    #namespace: bookthief
+ ```
+ 
+ Updated TrafficTarget spec with uncommented `Bookthief` kind:
+ ```
+ kind: TrafficTarget
+apiVersion: access.smi-spec.io/v1alpha2
+metadata:
+  name: bookstore-v1
+  namespace: bookstore
+spec:
+  destination:
+    kind: ServiceAccount
+    name: bookstore-v1
+    namespace: bookstore
+  rules:
+  - kind: HTTPRouteGroup
+    name: bookstore-service-routes
+    matches:
+    - buy-a-book
+    - books-bought
+  sources:
+  - kind: ServiceAccount
+    name: bookbuyer
+    namespace: bookbuyer
+  - kind: ServiceAccount
+    name: bookthief
+    namespace: bookthief
+ ```
+
```

Comment:
Can we keep this in the manifests file and add comments there? Looks a bit out of place.

Reply:
The goal is for user new to the setup to understand what exactly is happening as they run the manual demo.

Reply:
Yes but if the manifests are consolidated, might as well keep it there and add comments in the manifests file. We should definitely be adding comments explaining those files.

Reply:
I'm fine with adding the comments to the manifest, but for going through the demo, most people are not going to de-rail from the setup just to read what's in the manifest. They will just create/apply it and move on to the next step. This is just a visual representation of the file change so the user knows they did it correctly.